backend/routes/indexer/nft.go

```diff
@@ -135,6 +136,36 @@ func processNFTMintedEvent(event IndexerEvent) {
 		return
 	}
 
+	// Create a NFT JSON metadata file
+	metadata := map[string]interface{}{
+		"name":        fmt.Sprintf("NFT #%d", tokenId),
+		"description": "This is an NFT minted on the blockchain.",
+		"image":       fmt.Sprintf("ipfs://%s", imageHashHex),
```

Comment:
Image links will be to our backend for now.

Reply:
@b-j-roberts please can you explain better what you want here?

Reply:
This field stores a url to the image we generate of the NFT, which is exposed on the backend through the `nft-images/nft-{token id}.png` route. We wont be using ipfs to store images, so the ipfs url is incorrect.